site/class-1-bar/maroeska/readme.md

```diff
@@ -0,0 +1,8 @@
+# Bar chart 
```

Comment:
This is starting to look like something, but you’re missing [some essential sections](https://github.com/cmda-fe3/course-17-18/blob/master/class-2.md#description).

Reply:
Looks great now!